src/cmd/create.go

```diff
@@ -502,6 +553,23 @@ func createHelp(cmd *cobra.Command, args []string) {
 	}
 }
 
+// ensureImageName translates an image URI into a "pullable" reference
+//
+// On error is returned an empty string
+func ensureImageName(image, release string) string {
+	hasDomain := utils.ImageReferenceHasDomain(image)
+
+	if hasDomain {
+		return image
+	}
+
+	image, err := utils.GetFullyQualifiedImageFromDistros(image, release)
+	if err != nil {
+		return ""
```

Comment:
We shouldn't cut out the `error` from inner utility functions. Generally speaking, it's a good convention to always offer an `error` from functions that can possibly fail. It provides an explanation for the failure, which can be really useful when debugging.

Sometimes it can happen that an `error` thrown by one of the lower layers doesn't make sense in the context of the upper layers. In such cases, the `error` can either be wrapped or re-written, but an `error` should still be there.

Reply:
Would a debug print be enough? Or you'd rather have the function to return the error?

Reply:
In this case, I'd return the error, because it's directly tied to the success or failure of this function.

Sometimes, the success or failure of inner functions don't reflect directly on the function itself. In such cases, one could dump the error into the debug logs, if the error is being ignored or worked around.